### DISCUSSION TITLE
Override table output

### DISCUSSION THREAD

**Discussion by VoDmAl** [sulu/sulu#7065] [Q&A] [answered]
Hello!

Could you please help me with this?
Is it possible to override the output of the table when displaying content? I see that now the table created by the standard editor in admin is rendered as this HTML:
```
<figure class="table">
<table>
...
</table>
</figure>
```
I would like to change this and add some additional CSS classes. 

Thank you!

**Comment by alexander-schranz** [comment, marked as answer]
@VoDmAl Have a look at the following twing twig extension: https://github.com/sulu/web-twig/blob/2.5/docs/editor.md which allows you to add css classes to tags to that output. `{{ content.someEditor|editor_classes({table: 'special-table'}) }}`

Still if you are control of your CSS you can also just do:

```twig
<div class="editor"> 
    {{ content.someEditor|raw }}
</div>
```

And then create css with:


```css
.editor table {
     // ..
}
```

To add some additional CSS styles.

**Reply by VoDmAl** [reply]
@alexander-schranz Thank you very much for the solution. 

In general, this solves the problem. Do I understand correctly that specifying CSS in the admin panel is contrary to the spirit of SULU, because this is already a presentation layer? Therefore, such a solution, in fact, with a search and replay is not a hack, although it initially seems so?

**Reply by alexander-schranz** [reply]
Yeah this goes in the output of presentation and so it should live inside twig a twig extension. As they tags are very limited I don't see any problems with the search and replace aslong as ckeditor does not do any changes. Still not build up on that is mostly the best solution if you just do some parent class and handle all over CSS instead have to change the output.
